### DISCUSSION TITLE
Upload File

### DISCUSSION THREAD

**Discussion by ccfiel** [hayes/pothos#1011] [Q&A] [answered]
Nexus has upload file feature https://the-guild.dev/graphql/yoga-server/docs/features/file-uploads#usage-with-graphql-nexus

Do pothos has also upload file feature?

**Comment by hayes** [comment, marked as answer]
https://github.com/dotansimha/graphql-yoga/blob/main/examples/file-upload-nextjs-pothos/schema.ts

**Reply by ccfiel** [reply]
@hayes thanks a lot 👍
